### ISSUE TITLE
Very slow startup times when viewing image on NAS

### ISSUE THREAD

**Issue by Thane5** [easymodo/qimgv#383]
It seems qimgv has issues loading images from NAS servers. 
I have a couple thousand photos on one, and when i double click on one of them, it takes qimgv roughly ten seconds to display it.

Interestingly, while i am still waiting for to load, i can easily open the same photo with another viewer like Nomacs, which opens it almost instantly.

**Comment by tomasklaen** [comment]
Yeah I have the same issue with big folders. It seems qimgv won't start until it serializes the whole folder, which might be slow. It'd be way more usable to just display the image so that the startup is almost instant, and only than start preparing navigation stuff in the background. Maybe display some spinner to indicate that navigation is not ready yet.

**Comment by easymodo** [comment]
>just display the image so that the startup is almost instant, and only than start preparing navigation stuff in the background

I will do exactly that, yes. It's kinda noticeable on local folders as well

**Comment by CrypticCus** [comment]
I would also like this solution, i experience it even with a folder contains 2k photos in it.

**Comment by syaofox** [comment]
一样的问题
